client/src/models/populate_services.js

```diff
@@ -0,0 +1,215 @@
+import { get_client } from '../graphql_utils.js';
+import gql from 'graphql-tag';
+import { log_standard_event } from '../core/analytics.js';
+import { 
+  Service,
+  ServiceStandard,
+} from './services.js';
+
+
+const dept_service_fragment = (
+  `org_id
+      services: services {
+        service_id
+        program_ids
+        is_active
+
+        name
+        description
+        service_type
+        scope
+        target_groups
+        feedback_channels
+        urls
+        comment
+
+        last_gender_analysis
+
+        collects_fees
+        cra_buisnss_number_is_identifier
+        sin_is_identifier
+        account_reg_digital_status
+        authentication_status
+        application_digital_status
+        decision_digital_status
+        issuance_digital_status
+        issue_res_digital_status
+        digital_enablement_comment
+
+        telephone_enquires
+        website_visits
+        online_applications
+        in_person_applications
+        mail_applications
+        other_channel_applications
+
+        standards {
+          standard_id
+          service_id
+          is_active
+      
+          name
+      
+          last_gcss_tool_year
+          channel
+          standard_type
+          other_type_comment
+      
+          target_type
+          lower
+          upper
+          count
+          met_count
+          is_target_met
+          target_comment
+          urls
+          rtp_urls
+        }
+      }
+  `
+);
+
+const get_dept_services_query = gql`
+query($lang: String!, $id: String) {
+  root(lang: $lang) {
+    org(org_id: $id) {
+      id
+      ${dept_service_fragment}
+    }
+  }
+}
+`;
+
+const get_all_services_query = gql`
+query($lang: String!) {
+  root(lang: $lang) {
+    orgs {
+      ${dept_service_fragment}
+    }
+  }
+}
+`;
```

Comment:
Nitpick: these aren't get functions anymore, they directly contain the query values now.

Reply:
yeah good point :+1: